rocket_twin/tuto_marc_val/pipe_flow.py

```diff
@@ -0,0 +1,45 @@
+from cosapp.base import System, Port
+from cosapp.drivers import RungeKutta
+from cosapp.recorders import DataFrameRecorder
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+class connections_pipes(Port):
```

Comment:
class name should be with Capital letter

Reply:
ok